fuse/dl/models/model_multihead.py

```diff
@@ -64,6 +64,10 @@ def __init__(
         self.add_module("heads", self.heads)
 
     def forward(self, batch_dict: NDict) -> Dict:
+        if isinstance(self.conv_inputs, str):  # no number of input channels specified
```

Comment:
Few comments:
* self.conv_inputs is always a sequence (or None).
   You should check about the type of an element in this sequence instead. 
* To keep the code simple let's convert the sequence of a tuple to a sequence of str instead (keep only the keys)
* Let's move this code to the constructor - to do it only once

Reply:
I addressed most. can you clarify what you mean in the 2nd bullet?